webapp/javascript/pages/TagExplorerView.tsx

```diff
@@ -39,6 +39,13 @@ import { PAGES } from './constants';
 
 import styles from './TagExplorerView.module.scss';
 
+const getTimelineColorFromIndex = (index: number): Color => {
```

Comment:
```suggestion
const getTimelineColor = (index: number): Color => {
```
Since a) we use typescript so Intellisense can be assumed and b) there's already a parameter named index

Reply:
also let's receive a Palette declaratively

Reply:
im i got it right or what exactly do you mean by that ?

Reply:
I think he means lets pass the Palette into the function

Reply:
and also I think he means to remove `Index` from the funciton name since it's implied by the fact that you pass an index into the function